app/pages/about/publications-page.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import Publications from '../../lib/publications';
+import counterpart from 'counterpart';
+import Loading from '../../components/loading-indicator';
+
+export default class PublicationsPage extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      currentSort: 'showAll',
+    };
+    this.showPublicationsList = this.showPublicationsList.bind(this);
+
+  }
+
+  componentDidMount() {
+    this.loadProjects();
+  }
+
+  showPublicationsList(key) {
+    this.setState({currentSort: key});
+  }
+
+  projectSlugs() {
+    const slugs = [];
+    Object.keys(Publications).forEach((category) => {
+      Publications[category].forEach((project) => {
+        slugs.push(project.slug);
+      });
+    });
+    return slugs;
+  }
+
+  loadProjects() {
+    const slugs = this.projectSlugs();
+    const page_size = slugs.length;
+    apiClient.type('projects')
+        .get({
+          slugs,
+          cards: true,
+          page_size
+        }).then((result) => {
+          const projects = {};
+          result.forEach((project) => {
+            projects[project.slug] = project;
+          });
+          this.setState({
+            projects
+          });
+        });
+  }
+
+  renderSideBarNav(sideBarNav) {
+    return (
+      <nav>
+        { Object.keys(sideBarNav).map((navItem) => {
```

Comment:
I think there are some indenting, quotes and other little things here a little off, maybe:
```
{Object.keys(sideBarNav).map(navItem => (
  <button
    key={navItem}
    style={this.state.currentSort === navItem ? { fontWeight: 700 } : null}
    onClick={() => this.showPublicationsList(navItem)}
    className={'secret-button side-bar-button'}
  >
    <span>{sideBarNav[navItem]}</span>
  </button>
))}
```